modules/apps/adaptive-media/adaptive-media-image-impl/src/main/java/com/liferay/adaptive/media/image/internal/validator/AMImageValidatorImpl.java

```diff
@@ -95,6 +122,12 @@ public boolean isValid(FileVersion fileVersion) {
 		return true;
 	}
 
+	protected void setAMImageEntryLocalService(
```

Comment:
It's more of my taste to use ReflectionTestUtil than create a method only for testing, but LGTM